nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -896,6 +893,22 @@ in
           '';
       };
 
+      typesHashMaxSize = mkOption {
+        type = types.ints.positive;
+        default = if cfg.defaultMimeTypes == "${pkgs.mailcap}/etc/nginx/mime.types" then 2688 else 1024;
```

Comment:
I think we should move this into a let in to prevent it accidentally getting out of date.

Reply:
The mime.types file is rarely updated. If it is, the test will show that a change will be required.